lib/Onyx.js

```diff
@@ -757,6 +701,13 @@ function init({
         applyDecorators();
     }
 
+    if (maxCachedKeysCount > 0) {
+        setInterval(() => {
+            // Try to free anything dated from cache
+            cache.removeLeastRecentUsedKeys(maxCachedKeysCount);
+        }, 10 * 1000);
```

Comment:
Instead of interval based solution perhaps we can hook to something that happens less frequently
I think it would be OK to clean cache every minute or even less frequently as long as we clean it

Reply:
I feel maybe the interval based solution is too aggressive since most of the time we don't really need to clean the cache at all.

What if we hook the cache clean into the connection of a safe eviction key like the `reportActions_*`. If we are connecting to another large object that seems like a good opportunity to clean the cache in preparation.

Reply:
Thanks for the idea, that sounds much better, will update